src/js/map.js

```diff
@@ -0,0 +1,86 @@
+mapboxgl.accessToken = 'pk.eyJ1IjoicmZjeCIsImEiOiJoMEptMnlJIn0.LPKrjG_3AeYB5cqsyLpcrg';
```

Comment:
Is this safe to be in the js code? @rassokhin-s

Reply:
It's a public token, so yes. Even if we were receiving it from the backend - it would be a way to steal it.

Reply:
Ok thanks @rassokhin-s